webpack/scenes/ContentViews/Details/Repositories/ContentViewRepositories.js

```diff
@@ -60,6 +76,62 @@ const ContentViewRepositories = ({ cvId }) => {
   ];
   const loading = status === STATUS.PENDING;
 
+  const toggleBulkAction = () => {
+    setBulkActionOpen(!bulkActionOpen);
+  };
+  const onAdd = (repos) => {
+    const { repository_ids: repositoryIds } = details;
+    dispatch(updateContentView(cvId, { repository_ids: repositoryIds.concat(repos) }));
+  };
+
+  const onRemove = (repos) => {
+    const reposToDelete = [].concat(repos);
```

Comment:
This is valid, but `const reposToDelete = [...repos];` is the more 'modern' way to copy an array

Reply:
So repos param here can be either a single repo or array of repos coming from bulk remove..

Reply:
concat makes more sense then, thanks!